server/routes/auditRoutes.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-var audit = require('./../domain/audit');
+var audit = require('../domain/audit');

```

Comment:
FWIW we need to untangle etcD and the Audit code before splitting out any other providers - it makes it challenging to add a new one